.travis.yml

```diff
@@ -1,31 +1,25 @@
 ---
 dist: focal
 language: python
-python: "3.7"
+python: "3.9.15"
 os: linux
 services:
   - docker
 install:
-  - pip install pipenv
+  - pip3 install pipenv
```

Comment:
we could avoid this change by upgrading the travis dist to jammy (although I don't know if it is already supported in travis)

Reply:
Changed. Thanks.

Reply:
It doesn't work because cgroup v2 docker, so I still use focal dist.